internal/store/statefulset.go

```diff
@@ -193,6 +193,30 @@ func statefulSetMetricFamilies(allowAnnotationsList, allowLabelsList []string) [
 				}
 			}),
 		),
+		*generator.NewFamilyGeneratorWithStability(
+			"kube_statefulset_persistent_volume_claim_retention_policy",
```

Comment:
```suggestion
			"kube_statefulset_persistentvolumeclaim_retention_policy",
```
Since we are talking about the `PersistentVolumeClaim` kubernetes resource, it should be put in one word.

Reply:
done